apps/functions-subscription/src/adapters/azure/cosmosdb/activation-request.ts

```diff
@@ -77,7 +102,32 @@ export const makeActivationRequestReaderWriter = (
             // batches of 99 items.
             // https://learn.microsoft.com/en-us/javascript/api/@azure/cosmos/items?view=azure-node-latest#@azure-cosmos-items-batch
             RA.chunksOf(99)(rnea),
-            RA.map(makeBatchOperations(RNEA.head(rnea).trialId)),
+            RA.map(makeBatchOperations(RNEA.head(rnea).trialId, 'ACTIVE')),
+            TE.traverseArray((chunk) =>
+              TE.tryCatch(
+                () =>
+                  container.items.batch([...chunk], RNEA.head(rnea).trialId),
+                E.toError,
+              ),
+            ),
+            TE.map(RA.flatMap(({ result }) => result || [])),
+          ),
+        ),
+        O.getOrElseW(() => TE.of([])),
+        TE.map(makeActivationResult),
+      ),
+    // This method should decrease the counter of the job by the number of activationRequests with state = "ACTIVE"
+    updateActivationRequestsState: (activationRequests, state) =>
```

Comment:
**note**: maybe we could use this method to deprecate the `activate` one

Reply:
The `activate` method has been removed with commit [98c01ac](https://github.com/pagopa/trial-system/commit/98c01acee800407581630b7d6878caf750eabecf)